source/hic_hal/freescale/k20dx/gpio.c

```diff
@@ -55,7 +55,11 @@ void gpio_init(void)
     LED_CONNECTED_GPIO->PCOR  |= 1UL << LED_CONNECTED_BIT;
     // reset button configured as gpio input
     PIN_nRESET_GPIO->PDDR &= ~PIN_nRESET;
+#if defined(RBL_nRF51)
+    PIN_nRESET_PORT->PCR[PIN_nRESET_BIT] = PORT_PCR_MUX(1) | PORT_PCR_PE_MASK | PORT_PCR_PS_MASK;
```

Comment:
Looks like the RBL nRF51 has a hardware pullup on the reset button, so this line isn't needed.